java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -323,4 +327,16 @@ public CompletableFuture<Long> scard(String key) {
     public CompletableFuture<Long> exists(@NonNull String[] keys) {
         return commandManager.submitNewCommand(Exists, keys, this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<String> configRewrite() {
```

Comment:
Please remove from BaseClient and move to Standalone client. 
`configRewrite` without a route is only called in the Standalone client.

Reply:
nevermind. I was wrong